frontend/src/component/common/ApiTokenTable/ApiTokenTable.tsx

```diff
@@ -27,7 +9,7 @@ import { ConditionallyRender } from 'component/common/ConditionallyRender/Condit
 
 import { useConditionallyHiddenColumns } from 'hooks/useConditionallyHiddenColumns';
 
-const hiddenColumnsSmall = ['Icon', 'createdAt'];
+const hiddenColumnsMedium = ['Icon', 'createdAt', 'seenAt'];
 const hiddenColumnsCompact = ['Icon', 'project', 'seenAt'];
```

Comment:
What's the difference between medium/small screens and compact? Is that how they're rendered? I'm curious because they have almost the same columns hidden, but not quite.

Reply:
"compact" is here to indicate different set of columns for API tokens table inside Project settings. That's how it was done before so I didn't change it.

Reply:
Cool; all good.